### DISCUSSION TITLE
Example for plan diffuser

### DISCUSSION THREAD

**Discussion by JasperStolte** [opendilab/DI-engine#772] [Q&A] [answered]
Hi all,

I was looking through the examples and can't find one for Diffuser (which appears to be called plan diffuser in DI-engine).
Does anyone have a working example for this algorithm?

Cheers!

**Comment by PaParaZz1** [comment, marked as answer]
You can refer to this [config](https://github.com/opendilab/DI-engine/blob/main/dizoo/d4rl/config/halfcheetah_medium_pd_config.py) and [entry](https://github.com/opendilab/DI-engine/blob/main/dizoo/d4rl/entry/d4rl_pd_main.py) about Diffuser on the D4RL (MuJoCo) environment.
